changelog.md

```diff
@@ -277,6 +277,7 @@
 - Tuple expressions are now parsed consistently as
   `nnkTupleConstr` node. Will affect macros expecting nodes to be of `nnkPar`.
 
+- `nimscript` now accepts arbitrary file extensions.
```

Comment:
IMO we should now also deprecate `nim main` ? (without any any command nor `--eval`)
it's not particularly useful when `nim e main` works, and can now cause ambiguities.

should be done in a separate PR though.